delivery_carrier_label_batch/wizard/generate_labels.py

```diff
@@ -0,0 +1,279 @@
+# Copyright 2013-2019 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+import queue
+import logging
+import odoo
+import threading
+import codecs
+from contextlib import contextmanager
+from itertools import groupby
+from odoo import _, api, exceptions, fields, models, tools
+
+from ..pdf_utils import assemble_pdf
+from ..zpl_utils import assemble_zpl2
+
+_logger = logging.getLogger(__name__)
+
+
+class DeliveryCarrierLabelGenerate(models.TransientModel):
+
+    _name = 'delivery.carrier.label.generate'
```

Comment:
@tschanzt add `_description = "Generate labels from batch pickings"` to make runbot happy :)